app/bundles/PointBundle/Views/Point/form.html.php

```diff
@@ -29,6 +29,7 @@
 				    <?php
                     echo $view['form']->row($form['name']);
                     echo $view['form']->row($form['description']);
+                    echo $view['form']->row($form['scoringCategory']);
```

Comment:
let's re-arrange this UI a bit ![2017-10-06_18-26-30](https://user-images.githubusercontent.com/63312/31302006-0992bf60-aac4-11e7-8f7d-78c247f3a8fa.png)

Reply:
the same as before :

Don't know how to do that, maybe it is a 5 minute topic ?
In this case, could you maybe finish to polish the UI the way you expected to have it ?